### PR TITLE
[Stable10] Token polish

### PR DESCRIPTION
**Pull request by rullzer**
Backport of #806 
For #735 

CC: @MorrisJobke @LukasReschke

### CODE REVIEW

File: settings/templates/personal.php [hunk 1]
```diff
@@ -200,14 +200,18 @@
 		<button id="add-app-password" class="button"><?php p($l->t('Create new app password')); ?></button>
 	</div>
 	<div id="app-password-result" class="hidden">
-		<span><?php p($l->t('Use the credentials below to configure your app or device.')); ?></span>
+		<span>
+			<?php p($l->t('Use the credentials below to configure your app or device.')); ?>
+			<?php p($l->t('For security reasons this password will only be shown once.')); ?>
```
Comment (ChristophWurst) [inline settings/templates/personal.php]: A general questions: do we backport translations too? E.g. this string was probably not in NC10 before, hence it won't be translated properly

Comment (MorrisJobke) [inline settings/templates/personal.php]: > A general questions: do we backport translations too? E.g. this string was probably not in NC10 before, hence it won't be translated properly

I plan to fix this behavior soon ;)

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**